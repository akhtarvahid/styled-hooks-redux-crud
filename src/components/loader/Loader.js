import React from "react";
import "./loader.scss";

function Spinner() {
  return (
    <div className="root">
      <div className="root_loader"></div>
    </div>
  );
}

export default Spinner;
