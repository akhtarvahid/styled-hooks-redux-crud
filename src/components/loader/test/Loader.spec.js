import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../Loader';

test('should render Loader Component', () => {
    const tree = renderer
        .create(<Loader />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
