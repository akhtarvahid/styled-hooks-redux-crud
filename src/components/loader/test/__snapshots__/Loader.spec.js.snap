// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loader Component 1`] = `
<div
  className="root"
>
  <div
    className="root_loader"
  />
</div>
`;
